packages/es-components/src/components/base/icons/Icon.specs.js

```diff
@@ -10,5 +12,10 @@ it('includes the aria-hidden attribute', () => {
   const hasAriaHiddenAttribute = container
     .querySelector('i')
     .getAttribute('aria-hidden');
+  // render(<Icon name="federal" size={16} />);
+  // const hasAriaHiddenAttribute = screen
+  //   .getByRole('img')
+  //   .getAttribute('aria-hidden');
```

Comment:
I don't think we need any of the changes to this file. (It won't fail a lint if there are no changes in it.)